### ISSUE TITLE
Missing interior brim.

### ISSUE THREAD

**Issue by mat813** [slic3r/Slic3r#4714]
**Describe the bug**
The interior brim is not generated properly.

**To Reproduce**
Preview the stl with the config provided.

**3D Model and Slic3r Configuration Export**
[Archive.zip](https://github.com/slic3r/Slic3r/files/2788339/Archive.zip)

**Expected behavior**
There should be interior brim around all the "inside" stars.

**Screenshots**
![capture d ecran 2019-01-23 a 18 02 18](https://user-images.githubusercontent.com/75783/51623591-f8870400-1f30-11e9-96de-768e41c72e25.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version 916c30c

**Additional context**
May be related to #3993 but I am not sure.

**Comment by lordofhyphens** [comment]
Are the interior stars part of the same model?

**Comment by mat813** [comment]
Yes, the "stars" branches connect with each others and up to the sides.

**Comment by mat813** [comment]
Like this:

<img width="702" alt="screenshot 2019-01-23 a 23 40 54" src="https://user-images.githubusercontent.com/75783/51642088-6569c080-1f68-11e9-8fed-82702958f9b7.PNG">
